src/client.ts

```diff
@@ -0,0 +1,32 @@
+
+import axios from "axios"
+import datos from "./datos.json"
+
+
+async function postData() {
+  axios.post('http://95.217.235.69/',datos)
```

Comment:
```suggestion
  axios.post('http://95.217.235.69/', datos)
```

ya instalaste `prettier`?